storage/src/main/java/tech/pegasys/teku/storage/store/Store.java

```diff
@@ -391,6 +394,16 @@ public Optional<Bytes32> getProposerBoostRoot() {
     }
   }
 
+  @Override
+  public Set<UInt64> getEquivocatingIndices() {
+    readLock.lock();
+    try {
+      return equivocatingIndices;
```

Comment:
This isn't thread safe because the returned `equivocatingIndices` set will be accessed from outside the readLock. We'd have to return a copy of the set to make it thread safe. That's probably reasonable given the size should be quite limited.